templates/categories/macros.html

```diff
@@ -45,9 +45,15 @@
 
 <div class="ui card">
     {% if item.image %}
+        {% if primary_url %}
+            <a href="{{ primary_url }}">
+        {% endif %}
         <div class="image">
             <img src="{{ item.image }}" alt="Picture of {{ name }}">
         </div>
+        {% if primary_url %}
+            </a>
+        {% endif %}
```

Comment:
Due to the way Semantic UI works (i.e: badly), the anchor tag needs to replace the `div` rather that surround it.

```suggestion
         {% if primary_url %}
            <a class="image" href="{{ primary_url }}">
        {% else %}
            <div class="image">
        {% endif %}
            <img src="{{ item.image }}" alt="Picture of {{ name }}">
        {% if primary_url %}
            </a>
        {% else %}
            </div>
        {% endif %}
```